roles/edpm_nova/molecule/default/prepare.yml

```diff
@@ -154,3 +154,12 @@
         path: /etc/hosts
         line: "{{ ansible_default_ipv4.address | default('127.0.0.1') }} {{ ansible_fqdn }} {{ inventory_hostname }}"
         state: present
+
+    - name: Create config_file_for_delet file
```

Comment:
but you just create a file named 'test'. So I would simply call this: Create a test file to be deleted

Reply:
done